src/v2/Components/ArtworkFilter/ArtworkFilterContext.tsx

```diff
@@ -183,6 +183,8 @@ export interface ArtworkFilterContextProps {
   /** The current artwork filter state (which determines the network request and the url querystring) */
   filters?: ArtworkFiltersState
 
+  defaultFilters?: ArtworkFiltersState
```

Comment:
This is necessary in order to correctly restore the `sort` value depending on the A/B test when the "Clear All" button was pressed on mobile device

### Before
https://user-images.githubusercontent.com/3513494/157660295-3613af21-0a47-41c5-a2ea-1d9324ec3bf4.mp4

### After
https://user-images.githubusercontent.com/3513494/157660376-9bb51f2b-1b63-4b3b-838c-83a0975f96ca.mp4